src/client.ts

```diff
@@ -210,8 +209,11 @@ export class ScrapflyClient {
       throw new errors.ApiHttpClientError(JSON.stringify(data));
     }
 
-    const content = await this.handleLargeObjects(data.result)
-    data.result = content
+    const content_format = data.result.format
+    if (content_format === 'clob' || content_format === 'blob') {
+      const content = await this.handleLargeObjects(data.result, content_format)
```

Comment:
this can be simplified 

```suggestion
      data.result = await this.handleLargeObjects(data.result, content_format)
```